src/plots/piePlot.ts

```diff
@@ -275,11 +331,54 @@ namespace Plottable.Plots {
       let sectorValueAccessor = Plot._scaledAccessor(this.sectorValue());
       let dataset = this.datasets()[0];
       let data = this._getDataToDraw().get(dataset);
-      let pie = d3.layout.pie().sort(null).value((d, i) => sectorValueAccessor(d, i, dataset))(data);
+      let pie = d3.layout.pie().sort(null).startAngle(this._startAngle).endAngle(this._endAngle)
+        .value((d, i) => sectorValueAccessor(d, i, dataset))(data);
       this._startAngles = pie.map((slice) => slice.startAngle);
       this._endAngles = pie.map((slice) => slice.endAngle);
     }
 
+    private _pieCenter(): Point {
+      let a = this._startAngle < this._endAngle ? this._startAngle: this._endAngle;
```

Comment:
Sure, I'll add some comments. 
I've been trying to come up with a more generic way to compute the center but I haven't been able to find anything that doesn't leave out some possibilities potentially cutting out part of the pie.